devicetypes/heltun/he-relay.src/he-relay.groovy

```diff
@@ -0,0 +1,38 @@
+/**
+ * 
+ *
+ *  Copyright 2021 Sarkis Kabrailian
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ */
+metadata {
+	definition (name: "HE-RELAY", namespace: "HELTUN", author: "Sarkis Kabrailian", cstHandler: true, ocfDeviceType: "oic.d.switch") {
```

Comment:
If you're already doing a separate child DTH and they're not components (so they get their own device cards) would it make sense to break out all the individual switches' specific preferences into this child DTH to reduce the clutter of the main device?

They could call a parent method and pass their device ID and preference index and the parent could handle those calls by adding an endpoint-correct modifier to the index to handle them correctly.

Just a suggestion, by no means mandatory.

Reply:
I would prefer to keep it this way ))